src/sentry/sentry_metrics/indexer/base.py

```diff
@@ -22,6 +22,7 @@ class FetchType(Enum):
     HARDCODED = "h"
     DB_READ = "d"
     FIRST_SEEN = "f"
+    RATE_LIMITED = "r"
```

Comment:
The last seen updater only looks for DB_READ (`d`) and as far as I can tell doesn't ever enumerate over the fetch types.

But I do think this would break raw value storage in the snuba consumers. I can probably add a fix before any customer actually trips it, though:

https://github.com/getsentry/snuba/blob/3869db9a89d3e2e1922d464fec01634530d72439/snuba/datasets/generic_metrics_processor.py#L72-L78